torch/csrc/autograd/VariableTypeManual.cpp

```diff
@@ -235,6 +235,57 @@ Tensor & detach_(c10::DispatchKeySet ks, Tensor & self) {
   return self;
 }
 
+at::Tensor _noop_unary_manual(c10::DispatchKeySet ks, const at::Tensor & self) {
```

Comment:
What's the point of having these? They look like they're going to become stale very quickly as it is easy not to update them.

Reply:
They're probably manual because of the soundness problem in https://github.com/pytorch/pytorch/pull/59389/files#r645081077 :)

Reply:
I included manual versions because they provide a good way to both see what the kernel is doing and hack on it without having to dive into the codegen. (And rebuilding one cpp file is much faster than the ~700 that get recompiled when you touch codegen.) Staleness is a concern, but I think it should be manageable with proper CI. But I'm open to removing them if folks prefer that.

Reply:
Staleness is a double whammy because if things go stale (but not stale in a functional correctness way), you'll be benchmarking the wrong thing, *silently*